helm/ffc-demo-claim-service/templates/deployment.yaml

```diff
@@ -63,7 +63,7 @@ spec:
             - name: POSTGRES_DB
               value: {{ .Values.postgresDatabase }}
             - name: POSTGRES_HOST
-              value: {{ .Values.postgresHost }}
+              value: {{ .Values.postgresExternalName }}
```

Comment:
Shouldn't this still be `postgresHost`?

There's a Kubernetes [mapping external services](https://cloud.google.com/blog/products/gcp/kubernetes-best-practices-mapping-external-services) guide which suggests (in its second example) that an external service enables pods to call external resources using the external service's `metadata.name`, which effectively passes requests through to `spec.externalName`. In [our case](https://github.com/DEFRA/ffc-demo-claim-service/blob/jenkins/helm/ffc-demo-claim-service/templates/service.yaml), we set `metadata.name` to `.Values.postgresHost` and `spec.externalName` to `.Values.postgresExternalName`.

Reply:
That's odd, the payment service was switched to use `postgresExternalName` when the [Jenkins build pipeline](https://github.com/DEFRA/ffc-demo-payment-service/commit/9f1dc3dafcc176b279e458e01b0cda8b70083e4f) was added. The user service still uses `postgresHost`.

@johnwatson484 Do you recall the reason for changing the payment service `POSTGRES_HOST` value?

Reply:
Looking back at the `development-values.yaml`, it's clear that it should be `postgresHost`. The reason we have them as two variables is so that in production-like deployments we can use `postgresExternalName` to direct requests to an external database, whereas in development we can leave `postgresExternalName` blank to prevent creation of the external service and direct requests within the Kubernetes network to the local postgres container.

Reply:
I'm satisfied that this has been reverted and a ticket added to the backlog to check the usage of `postgresHost` and `postgresExternalName`:

https://eaflood.atlassian.net/browse/PSD-227